resources/partial/notification-type-springboot/client-extension.yaml

```diff
@@ -0,0 +1,8 @@
+${id}-user-agent:
+  type: oauthApplicationUserAgent
+  scopes:
+    - Liferay.Notification.REST.everything
```

Comment:
here I think you can remove scopes entirely since you are not making calls back to DXP.

Reply:
Got it!

Thank you!